src/test/java/edu/indiana/dlib/amppd/service/AmpUserServiceTests.java

```diff
@@ -127,6 +127,20 @@ public void shouldActivateUser() throws Exception{
 		  }
     }
 	
+	@Test
+	public void shouldGetUserById() throws Exception{
+		AmpUser user = getAmpUser();
+		ampUserService.registerAmpUser(user);
+		try {
+			AmpUser fetchByUserName = ampUserRepository.findByUsername(user.getUsername()).get();
+			AmpUser fetchByUserId = ampUserService.getUserById(fetchByUserName.getId());
+			Assert.assertEquals(fetchByUserName,fetchByUserId);
+		  }
+		  catch(Exception ex) {
+			  System.out.println(ex.toString());
+		  }
+	}
+	
 	private AmpUser getAmpUser() {
```

Comment:
A few suggestions:
!) To fully test a method, we should try to cover various cases, including both positive and negative ones
2) For clarity, it's best to separate each test case into a different test method, and the test name should indicate what's being tested. For ex, shouldGetUserWithValidId, shouldReturnNullWithInvalidUserId, or shouldThrowExceptionWithInvalid***. 
3) You wouldn't want to catch exception here, since you are testing and want to expose exception if it happens.
4) If you do catch exception, you don't need to put throws Exception on the method signature. 
5) You can add one test shouldReturnNullWithInvalidUserId.